app/models/case_log.rb

```diff
@@ -460,10 +464,69 @@ def self.label_from_value(value)
   end
 
   def self.csv_attributes(user)
-    attributes = attribute_names - %w[owning_organisation_id managing_organisation_id created_by_id] + %w[unittype_sh owning_organisation_name managing_organisation_name created_by_name]
+    attributes = attribute_names + %w[unittype_sh]
+    attributes = replace_csv_id_fields_with_names(attributes)
+
+    attributes = order_csv_attributes(attributes)
+
     user.present? && !user.support? ? attributes - CSV_FIELDS_TO_OMIT : attributes
   end
 
+  def self.replace_csv_id_fields_with_names(initial_attributes)
+    attributes = initial_attributes.clone
+    { "managing_organisation_id": "managing_organisation_name", "owning_organisation_id": "owning_organisation_name", "created_by_id": "created_by_name" }.each { |current, new_attribute| attributes[attributes.index(current.to_s)] = new_attribute }
+    attributes
+  end
+
+  def self.order_csv_attributes(initial_attributes)
+    attributes = initial_attributes.clone
+    ordered_default_form_questions = move_checkbox_answer_options(FormHandler.instance.forms.first.second.questions)
```

Comment:
Should this always take the first form? What if the logs being downloaded are all for the second collection window?

Reply:
Yeah, I just went with the default. I'm not sure what it should do, this is the easiest if we assume that the 2 years questions are relatively similar. I'll spike some other options